src/textual_summary/stories/MultilinkTable.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import MultilinkTable from '../multilink_table';
+import { multilinkTableData } from '../data/multilink_table';
+
+storiesOf('TextualSummary', module)
+  .add('MultilinkTable', () => {
+    return (
+      <MultilinkTable
+        title={multilinkTableData.title}
+        items={multilinkTableData.items}
+        onClick={e => null}
```

Comment:
again `e => null`, not a fan

Reply:
The explanation remains the same here ;-)